client/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:lts-slim
+FROM node:16-slim
```

Comment:
Do we have specific dependencies on node16?

Reply:
The front-end needs NodeJS 14 to install, so I don't want the gap between installation and runtime too big.